journalbeat/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.10.3
```

Comment:
I think this docker file will become useful but I wonder if it is used in the tests or if it was missing before?

Reply:
It was missing before. I has every dependency installed, so everyone can build and run `journalbeat`.